hooks/post_gen_project.py

```diff
@@ -0,0 +1,28 @@
+import random
+import string
+import os
+
+secret = ''.join(
+    random.choice(string.printable[:94].replace('"', '')) for _ in range(50))
+
+dirs = [
+    f for f in os.listdir('.') if all(
+        a == b for a, b in zip(
+            os.path.basename(os.getcwd()).split('-'),
+            f.split('-')
+        )
+    )]
+
+assert len(dirs) == 1
+
+dirname = dirs[0]
```

Comment:
Dit soort top-level dingen vind ik toch iets netter in een functie. En dan misschien die functie a/h eind wèl aanroepen op het toplevel. Kan je het gelijk wat makkelijker testen.